ts/tree/leaves/Node.ts

```diff
@@ -0,0 +1,38 @@
+import { nodeDB } from "../../client/database/connection";
+import { Node } from "../hasher";
+
+export const getNodeKey = (
+    name: string,
+    depth: number,
+    index: number
+): string => `${name}${depth}${index}`;
+
+export class ItemNode {
+    static async fromDB(
+        name: string,
+        depth: number,
+        index: number
+    ): Promise<Node> {
+        const key = getNodeKey(name, depth, index);
+        return await nodeDB.get(key);
```

Comment:
`await`ing is not needed in async chains when returning values, see this discussion https://github.com/thehubbleproject/hubble-contracts/pull/556#discussion_r618636732
```suggestion
        return nodeDB.get(key);
```

Reply:
gotcha, thanks for the detailed explanation 😄